x/datadeal/keeper/deal.go

```diff
@@ -162,7 +162,7 @@ func (k Keeper) SellData(ctx sdk.Context, msg *types.MsgSellData) error {
 	}
 
 	getDataSale, _ := k.GetDataSale(ctx, msg.VerifiableCid, msg.DealId)
-	if getDataSale != nil && getDataSale.Status != types.DATA_SALE_STATUS_FAILED {
+	if getDataSale != nil && getDataSale.Status != types.DATA_SALE_STATUS_VERIFICATION_FAILED {
 		return sdkerrors.Wrapf(types.ErrSellData, "data already exists")
 	}
```

Comment:
Should we distinguish between failed and rejected in data verification?
IMHO, if the data is rejected in verification, seller cannot sell the same data. wdyt?

Reply:
On the other hand, if the verification of data is failed(due to the downtime of oracles), the seller can try to sell the same data again.

Reply:
Does it means the `REJECTED` and `FAILED` are different?
So, if the status is `FAILED`, seller can sell the same data again. Am I understand right?

Reply:
I have the question. So, when the `REJECTED` status need?
Does the `REJECTED` status will be added in `Endblocker`?

Reply:
> Does it means the REJECTED and FAILED are different?
So, if the status is FAILED, seller can sell the same data again. Am I understand right?

Right. If the result of vote is `No`(which may mean 2/3+ `No`?), the data verification is `REJECTED`.
And if there is no consensus value, the status of data sale might be `FAILED`.

Reply:
> I have the question. So, when the REJECTED status need?
Does the REJECTED status will be added in Endblocker?

If I am right and we need to distinguish between `FAILED` and `REJECTED`, we need to handle the result differently.

Reply:
Give me some seconds. I'll think more about it.

Reply:
It would be good to distinguish between `FAILED` and `REJECTED`, but I don't think it's necessary to distringuish them.
Because if oracle vote with `No` once, then oracle will continue to vote with `No`.
So, the seller just keeps paying the fee, and there seems to be no problem functionally.

Reply:
Right. I agree that it's not essential.
Let's keep this way for now.